README.md

```diff
@@ -53,34 +53,31 @@ A partial documentation is available at: https://mace-docs.readthedocs.io
 
 ## Installation
 
-### 1. Requirements:
+### 1. Requirements
 
 - Python >= 3.7  (for openMM, use Python = 3.9)
 - [PyTorch](https://pytorch.org/) >= 1.12 **(training with float64 is not supported with PyTorch 2.1 but is supported with 2.2 and later, Pytorch 2.4.1 is not supported)**
 
 **Make sure to install PyTorch.** Please refer to the [official PyTorch installation](https://pytorch.org/get-started/locally/) for the installation instructions. Select the appropriate options for your system.
 
 ### Installation from PyPI
+
 This is the recommended way to install MACE.
 
 ```sh
 pip install --upgrade pip
 pip install mace-torch
 ```
-**Note:** The homonymous package on [PyPI](https://pypi.org/project/MACE/) has nothing to do with this one.
 
+**Note:** The homonymous package on [PyPI](https://pypi.org/project/MACE/) has nothing to do with this one.
 
 ### Installation from source
 
-
 ```sh
 git clone https://github.com/ACEsuit/mace.git
 pip install ./mace
```

Comment:
replace that with
```
python3 -m pip install git+https://github.com/ACEsuit/mace.git
```

Reply:
why not go straight to the fastest python package manager?

```sh
uv pip install git+https://github.com/ACEsuit/mace
```

Reply:
because not everyone has uv installed but in a perfect world uv is better that just pip, faster.